core/esmf-aspect-model-resolver/src/main/java/org/eclipse/esmf/aspectmodel/resolver/AspectModelResolver.java

```diff
@@ -13,6 +13,9 @@
 
 package org.eclipse.esmf.aspectmodel.resolver;
 
+import static io.vavr.API.*;
```

Comment:
I propose to use single import without ".*"

But if you use all functionality from this package, don't need.

See for yourself